assets/templates/karma/karma.conf.js.tmpl

```diff
@@ -0,0 +1,36 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
+
+module.exports = function (config) {
+  config.set({
+    basePath: '',
+    frameworks: ['jasmine', '@angular-devkit/build-angular'],
+    plugins: [
+      require('karma-jasmine'),
+      require('karma-chrome-launcher'),
+      require('karma-jasmine-html-reporter'),
+      require('karma-coverage-istanbul-reporter'),
+      require('@angular-devkit/build-angular/plugins/karma'),
+      require('karma-spec-reporter'),
+    ],
+    client: {
+      clearContext: false, // leave Jasmine Spec Runner output visible in browser
+    },
+    coverageIstanbulReporter: {
+      dir: require('path')
+        .join(__dirname, 'coverage'),
+      reports: ['html', 'lcovonly', 'text-summary'],
+      fixWebpackSourcePaths: true,
+    },
+    angularCli: {
+      environment: 'dev',
+    },
+    reporters: ['spec', 'kjhtml'],
+    port: 9876,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: true,
+    browsers: ['Chrome'],
```

Comment:
Should we define also the Headless Chrome browser here? We usual tend to use it, so it would be nice if it is already defined. It will be use only if the user explicitly starts it with the new defined browser

Reply:
Added it, as it is not used by default but will be there already.